exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/validate/BatchValidator.java

```diff
@@ -320,6 +321,19 @@ private void validateNullableVector(String name, NullableVector vector) {
           "Outer value count = %d, but inner value count = %d",
           outerCount, valueCount));
     }
+    if (vector instanceof NullableVarCharVector) {
```

Comment:
Could you please add these checks for other nullable `VariableWidthVector` implementations: `NullableVarBinaryVector`, `NullableVar16CharVector` and `NullableVarDecimalVector`.